docs/rules/no-negated-ok.md

```diff
@@ -46,6 +46,12 @@ QUnit.test('test', function (assert) {
 
 ```
 
+## Configuration
+
+This rule takes an optional object containing:
+
+* `boolean` -- `fixToNotOk` -- (default false) whether the rule should autofix `assert.ok(!foo)` to `assert.notOk(foo)` ([notOk](https://api.qunitjs.com/assert/notOk/) was added in QUnit 1.18)
```

Comment:
Just to avoid potentially confusing people into thinking `boolean` is a property name, what do you think of this?

```suggestion
* `fixToNotOk` (boolean, default: false): Whether the rule should autofix `assert.ok(!foo)` to `assert.notOk(foo)` ([notOk](https://api.qunitjs.com/assert/notOk/) was added in QUnit 1.18)
```